tests/syscall/fs/host/host.c

```diff
@@ -3,59 +3,70 @@
 
 #if defined(_WIN32)
 #include <windows.h>
+#pragma comment(lib, "Shell32.lib")
+#else
+#include <dirent.h>
 #endif
 #include <openenclave/host.h>
 #include <openenclave/internal/syscall/host.h>
 #include <openenclave/internal/tests.h>
 #include <stdio.h>
+#include <string.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include "fs_u.h"
 
 #define SKIP_RETURN_CODE 2
 
-int rmdir(const char* path);
-
-int test_fs_posix(
-    const char* enclave_path,
-    const char* src_dir,
-    const char* tmp_dir)
+#if defined(_WIN32)
+int rmdir(wchar_t* path)
```

Comment:
Nice! It might be useful to name it something that indicates its function like `recursive_rmdir()` to differentiate its behavior from the standard `rmdir()`

We might also want to consider refactoring this out for other syscall tests to use (notably hostfs, which as the same structure) in a later PR.

Reply:
Fixed, thanks!